openreferee_server/notify.py

```diff
@@ -0,0 +1,45 @@
+import json
+import threading
+
+from .operations import setup_requests_session
+
+
+class NotifyService:
+    def __init__(self, url=None, token=None) -> None:
+        self.url = url
+        self.token = token
+        self.session = None
+
+    def send(self, payload):
+        try:
+            self.session.get(self.url, data=json.dumps({"payload": payload}))
+        except Exception:
+            pass
```

Comment:
No. Don't do that. At least print the exception and traceback if you decide to ignore it. But don't silently drop *any* exception. Also, be more specific (request failing, possibly a timeout as well) and don't catch just about anything. If there's let's say a TypeError (just an example) that'd be a bug and not something you want to silently ignore.